packages/block-editor/src/components/inserter/menu.js

```diff
@@ -404,7 +404,11 @@ export class InserterMenu extends Component {
 								</div>
 								<Tip>
 									{ __(
-										'While writing, you can press "/" to quickly insert new blocks.'
+										'While writing, you can press'
+									) }
+									<kbd className="edit-post-keyboard-shortcut-help__shortcut-key">/</kbd>
```

Comment:
The class name applied here is from another component, which isn't really something encouraged as it can lead to maintainability issues (i.e. in the future someone changes the other component and doesn't realise this one will be affected).

I think there are three options:
- Introduce a generic `<Keyboard>` react component in the `@wordpress/components` package that has this styling and can be use across the editor. There would have to be some thought about whether this is something wanted in that package.
- Introduce some global styling for the `kbd` element or a mixin. Not sure about this option as I think it's nice to keep the global styling to a minimum and promote using components instead.
- Duplicate some of the styles in tip component.

I think the third one might be preferable. As far as I know there are only two places in the codebase that have `kbd` styles. When there are three or four it might be time to consider refactoring to a component.